### PR TITLE
feat: stabilize 'alpnProtocols' setting

### PR DESCRIPTION
**Pull request by bartlomieju**
Ref https://github.com/denoland/deno/issues/19685

### CODE REVIEW

File: cli/tests/integration/lsp_tests.rs [hunk 1]
```diff
@@ -4788,7 +4788,7 @@ fn lsp_completions_auto_import() {
         "source": "./b.ts",
         "data": {
           "exportName": "foo",
-          "exportMapKey": "foo|6799|file:///a/b",
+          "exportMapKey": "foo|6802|file:///a/b",
```
Comment (dsherret) [inline cli/tests/integration/lsp_tests.rs:4791]: We should find a way to remove this from the test somehow lol

**Review by dsherret — APPROVED** [review]
LGTM!

File: cli/tests/integration/lsp_tests.rs (hunk 1 above)
Comment (bartlomieju) [reply cli/tests/integration/lsp_tests.rs:4791]: Yeah, it's been a PITA to update it every time :D

**Comment by avindra** [comment]
Getting an error about it being unstable in 1.35.0:

```
Unstable API Deno.listenTls#alpn_protocols. The --unstable flag must be provided.
```

**Comment by bartlomieju** [comment]
Ooops, thanks for pointing this out, I missed it. Fixed in https://github.com/denoland/deno/pull/19732.

**Pull request merged**